lightkurve/lightcurvecollection.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division, print_function
+
+import copy
+from tqdm import tqdm
+import os
+import datetime
+import logging
+from collections import Sequence
+
+import oktopus
+import numpy as np
+from scipy import signal
+from scipy.optimize import minimize
+from matplotlib import pyplot as plt
+import matplotlib as mpl
+
+from astropy.stats import sigma_clip
+from astropy.table import Table
+from astropy.io import fits
+from astropy.time import Time
+
+
+__all__ = ['LightCurve', 'KeplerLightCurve', 'TessLightCurve',
+           'iterative_box_period_search']
+
+log = logging.getLogger(__name__)
+
+class LightCurveCollection(Sequence):
+    """
```

Comment:
These docstrings need to be in the following format:
http://docs.astropy.org/en/stable/development/docrules.html#docstring-standard

Reply:
Done